backend/database/.gitignore

```diff
@@ -1,3 +0,0 @@
-* 
```

Comment:
why did you delete this?

Reply:
It's no longer necessary. Whenever we perform scraping, we create a /cache/database folder in the Docker container that will hold the SQLite scratchpad database. This folder is basically useless.